Fix Issue #4 for real

KeenIIDX opened the pull request:
The sample memory card has 2 saves for the same game, I think.  It also causes mymc to try and write a save file that has invalid characters in the file name.  This should take care of both issues.

There's probably a better way to detect mymc throwing the specific File Exists error, but I could not find documentation on how to make Powershell do that.

KeenIIDX committed:
Tidy up regex

KeenIIDX committed:
Fix Issue #4

Create unique file names for each save in a file exported.

K3zter commented:
Hey thanks for looking at this, had a lot of distractions lately but I will review this on Monday.

I meant to look at that issue sooner, I tried the attached file on my machine and it seemed to work fine! However I was using the psu_injection branch so I'm wondering if something I did there fixed it already.

KeenIIDX commented:
i wasn't super happy with this fix, just figured out a better one, i'll get that coded tonight.

KeenIIDX committed:
Redo fix again

I think this is a better fix than hoping the error that mymc was generating was always due to an existing file.

K3zter reviewed (approved):
Thanks my original regex was very ugly!

K3zter commented:
These changes make sense - using PowerShell's built-in invalid file name chars method is a great idea! I think the duplicate file issue could plausibly come up in regular scenarios where people are importing lots of saves (some people have loads of saves from the same game on different cards). Thanks for your help!
